File: uploads/ComplexOperations.java
public class ComplexOperations {
    public static void main(String[] args) {
        for (int i = 0; i < 10; i++) {
            if (i % 2 == 0) {
                System.out.println("Even: " + i);
            } else if (i % 3 == 0) {
                System.out.println("Divisible by 3: " + i);
            } else {
                System.out.println(i);
            }
        }
        System.out.println("Factorial of 5: " + factorial(5));
    }

    public static int factorial(int n) {
        if (n <= 1) {
            return 1;
        } else {
            return n * factorial(n - 1);
        }
    }
}

File: uploads/ComplexOperations.cpp
#include <iostream>
using namespace std;

int factorial(int n) {
    if (n <= 1) return 1;
    else return n * factorial(n - 1);
}

int main() {
    for (int i = 0; i < 10; i++) {
        if (i % 2 == 0) cout << "Even: " << i << endl;
        else if (i % 3 == 0) cout << "Divisible by 3: " << i << endl;
        else cout << i << endl;
    }
    cout << "Factorial of 5: " << factorial(5) << endl;
    return 0;
}
